src/graphql/value-to-ast.ts

```diff
@@ -0,0 +1,55 @@
+import { Kind, ObjectFieldNode, ValueNode } from 'graphql';
+
+export function createValueNodeFromValue(value: unknown): ValueNode {
```

Comment:
(optional)
if the file only contains a single method you could also name the file like the method (or the other way around) to make navigation easier

Reply:
I chose the name for consistency with `value-from-ast.ts`